sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/api/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.netstandard2.0.cs

```diff
@@ -60,6 +60,7 @@ public abstract partial class ActionableCloudEventResponse<T> : Microsoft.Azure.
     public abstract partial class ActionableResponse<T> : Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework.AuthenticationEventResponse where T : Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework.AuthenticationEventAction
     {
         protected ActionableResponse() { }
+        [System.ComponentModel.DataAnnotations.RequiredAttribute]
```

Comment:
I believe this would constitute a breaking change.   Previously, I could set the list to `null` and now I cannot. 

Please be sure to get architect sign-off from @KrzysztofCwalina on an API View since there are changes to the public API.

Reply:
Thanks Jesse, but as this is unreleased no one has used it per say, would I still need to go down this route of getting approval?